docs/tutorials/performance/load-test-grpc-multiple.md

```diff
@@ -2,14 +2,14 @@
 template: main.html
 ---
 
-# Load test multiple gRPC endpoints
+# Load test multiple gRPC methods
 
-[Load Test gRPC](./load-test-grpc.md) describes how to load test a single method from a gRPC service inside Kubernetes. This tutorial expands on the previous tutorial and describes how to load test multiple endpoints from an HTTP service.
+The tutorial [Single gRPC method](./load-test-grpc.md) describes how to load test a single method from a gRPC service inside Kubernetes. This tutorial expands on this tutorial and describes how to load test multiple endpoints from an HTTP service.
```

Comment:
```suggestion
The tutorial [Single gRPC method](./load-test-grpc.md) describes how to load test a single method from a gRPC service inside Kubernetes. This tutorial expands on that tutorial and describes how to load test multiple endpoints from an HTTP service.
```

"**This tutorial** expands on **this tutorial**" reads a bit strange

Reply:
rewritten and shortened.